src/GovUk.Education.ExploreEducationStatistics.Public.Data.Processor/Functions/StatusCheckFunction.cs

```diff
@@ -0,0 +1,33 @@
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Azure.Functions.Worker;
+using Microsoft.DurableTask.Client;
+
+namespace GovUk.Education.ExploreEducationStatistics.Public.Data.Processor.Functions;
+
+public class StatusCheckFunction
+{
+    private static readonly OrchestrationQuery ActiveOrchestrationsQuery = new()
+    {
+        Statuses = new List<OrchestrationRuntimeStatus>
+        {
+            OrchestrationRuntimeStatus.Pending,
+            OrchestrationRuntimeStatus.Running
+        }
```

Comment:
Nitpick, but could use a collection expression?

Reply:
All nitpicks welcome here!  Done - I'm almost surprised that Rider didn't suggest it also :)